test-suite/output/Int63Syntax.out

```diff
@@ -1,4 +1,4 @@
-2%int63
+2%uint63
```

Comment:
I don't know why `Set Printing All` picks `%int63` instead of `%uint63`. To be fair, if it'd picked `%uint63` I would still not know why. I tried changed the order in which the scopes are declared, but nothing changed.

Reply:
Ok, I had to change the default printer in `interp/constrextern.ml`.